backend/src/main/java/com/bakdata/conquery/models/forms/mostlyai/MostlyAiExecution.java

```diff
@@ -0,0 +1,266 @@
+package com.bakdata.conquery.models.forms.mostlyai;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStreamWriter;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import javax.ws.rs.BadRequestException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.StreamingOutput;
+
+import com.bakdata.conquery.apiv1.ExecutionStatus;
+import com.bakdata.conquery.apiv1.forms.MostlyAiForm;
+import com.bakdata.conquery.io.cps.CPSType;
+import com.bakdata.conquery.io.result.csv.CsvRenderer;
+import com.bakdata.conquery.io.result.external.ExternalResult;
+import com.bakdata.conquery.io.result.external.ExternalResultProcessor;
+import com.bakdata.conquery.io.result.external.ExternalResultProvider;
+import com.bakdata.conquery.io.storage.MetaStorage;
+import com.bakdata.conquery.models.auth.entities.Subject;
+import com.bakdata.conquery.models.auth.entities.User;
+import com.bakdata.conquery.models.config.CSVConfig;
+import com.bakdata.conquery.models.config.ConqueryConfig;
+import com.bakdata.conquery.models.datasets.Dataset;
+import com.bakdata.conquery.models.error.ConqueryError;
+import com.bakdata.conquery.models.execution.ExecutionState;
+import com.bakdata.conquery.models.execution.ManagedExecution;
+import com.bakdata.conquery.models.forms.managed.ManagedForm;
+import com.bakdata.conquery.models.i18n.I18n;
+import com.bakdata.conquery.models.identifiable.mapping.EntityPrintId;
+import com.bakdata.conquery.models.messages.namespaces.WorkerMessage;
+import com.bakdata.conquery.models.messages.namespaces.specific.ExecuteForm;
+import com.bakdata.conquery.models.query.ManagedQuery;
+import com.bakdata.conquery.models.query.PrintSettings;
+import com.bakdata.conquery.models.query.resultinfo.ResultInfo;
+import com.bakdata.conquery.models.query.resultinfo.UniqueNamer;
+import com.bakdata.conquery.models.query.results.EntityResult;
+import com.bakdata.conquery.models.worker.DatasetRegistry;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.google.common.collect.MoreCollectors;
+import it.unimi.dsi.fastutil.Pair;
+import kotlin.text.Charsets;
+import lombok.EqualsAndHashCode;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.FileUtils;
+
+@CPSType(id = "MOSTLY_AI_EXECUTION", base = ManagedExecution.class)
+@Slf4j
+@EqualsAndHashCode(callSuper = true)
+public class MostlyAiExecution extends ManagedForm implements ExternalResult {
```

Comment:
excuse me, but what is this and what does it do?

Reply:
Das ist eine Interface, dass ich aus EVA rüber gezogen habe.
Das erlaubt einer Execution selber den Body des Download-Requests zu streamen

Reply:
Sorry war etwas zu blöd formuliert, aber bitte dokumentier das hier